html-css/building-blocks/step4-instructions.md

```diff
@@ -1,8 +1,8 @@
-Demos show how PatternFly’s components and layouts can be put together to build more complex structures. Demos have no additional styling as they are strictly made from components and layouts. If styling is necessary to implement a desired demo, then new components or layouts, or variants of the components or layouts used, should be created instead.
+"Demos" show how PatternFly’s components and layouts can be put together to build more complex structures. Demos have no additional styling as they are strictly made from components and layouts. If styling is necessary to implement a desired demo, then new components or layouts, or variants of the components or layouts used, should be created instead.
```

Comment:
```suggestion
"Demos" show how PatternFly’s components and layouts are put together to build more complex structures. Demos have no additional styling as they are strictly made from components and layouts. If styling is necessary to implement the desired demo, then new components or layouts, or variants of the components or layouts used, should be created instead.
```

Reply:
Rewrite: "Demos" show how PatternFly’s components and layouts are put together to build more complex structures. Demos have no additional styling; they are strictly made from components and layouts. If styling is necessary to implement the desired demo, then instead create new components or layouts, or variants of the components or layouts. 

I don't know what we're saying here: "If styling is necessary to implement the desired demo." I think what we mean is "If styling is needed for a certain demo," but I didn't want to put that there in case we're meaning something else. Either way, let's clarify and make it sound less formal.